jbmc/src/java_bytecode/java_bytecode_concurrency_instrumentation.cpp

```diff
@@ -87,7 +87,7 @@ static const std::string get_thread_block_identifier(
   PRECONDITION(f_code.arguments().size() == 1);
   const exprt &expr = f_code.arguments()[0];
   const mp_integer lbl_id =
-    numeric_cast_v<mp_integer>(to_constant_expr(expr.op0()));
+    numeric_cast_v<mp_integer>(to_constant_expr(to_multi_ary_expr(expr).op0()));
```

Comment:
Slightly struggling with this one. `expr` is the first argument to a call to `startThread` but it is a `multi_ary_exprt`. What is it?

Reply:
I was massively struggling with this one, and gave up.